index.js

```diff
@@ -19,10 +19,11 @@ const retryableErrorCodes = new Set([
   'ENETUNREACH',
   'EAI_AGAIN'
 ]);
+const retryableErrorCodesArray = Array.from(retryableErrorCodes);
```

Comment:
I would just convert `retryableErrorCodes` to an array instead of using `Array.from()`, and the same below for status codes.

Reply:
Error codes have a different delay (1s) so there's a `retryableErrorCodes.has()` check. We can do `retryableErrorCodes.includes()` if we switch over to an array?